index.html

```diff
@@ -15,16 +15,171 @@
         <nav><i class="fa fa-bars"></i></nav>
     </header>
     <main>
+
+    <!-- THIS IS THE SECTION FOR MOBILE VERSION -->
+        <section class="grid-works">
+            <!-- CARD 1 (Tonic)-->
+            <div class="works-card1">
+                <div class="child-worksCard-info">
+                    <img src="./images/SnapshootPortfolio.png" alt="snapshoot">
+                    <h2 class="prjct-title">TONIC</h2>
+                    <div class="prjct-info">
+                        <h6 class="canopy client-name">CANOPY</h6>
+                        <div class="dot-separator"></div>
+                        <h6 class="client-name">Back End Dev</h6>
+                        <div class="dot-separator"></div>
+                        <h6 class="client-name">2015</h6>
```

Comment:
Your design is looking good 👍 so far, but there are a few changes to look into.

[ Required Changes ]

- [ ] Kindly move this section below your headline section  `<section class="hdln-section">`. The headline section is meant to be the first section in your portfolio that gives a brief description of you as a developer. Please fix that👍